e2e/LongHaul/service/Program.cs

```diff
@@ -81,9 +79,6 @@ await Task
                     .WhenAll(
                         systemHealthMonitor.RunAsync(cancellationTokenSource.Token),
                         iotHub.MonitorConnectedDevicesAsync(cancellationTokenSource.Token),
-                        iotHub.InvokeDirectMethodAsync(cancellationTokenSource.Token),
-                        iotHub.SetDesiredPropertiesAsync("guidValue", Guid.NewGuid().ToString(), cancellationTokenSource.Token),
-                        iotHub.SendC2dMessagesAsync(cancellationTokenSource.Token),
```

Comment:
Moved to "service/IotHub.cs", will be called from `MonitorConnectedDevicesAsync()`